lib/cli/src/crewai_cli/templates/AGENTS.md

```
@@ -645,7 +671,11 @@ def handle_create_video(self):
     ...
 ```
 
+<<<<<<< HEAD
```

markdownlint-cli2 (0.23.2)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)